src/main/java/com/dynatrace/jenkins/dashboard/model_2_0_0/TAReport.java

```diff
@@ -41,16 +41,16 @@
  */
 public class TAReport {
 
-	private final AbstractBuild<?, ?> build;
+	private final Run<?, ?> build;
```

Comment:
rename build -> run

Reply:
Renaming these variables requires a lot of changes in .jelly files .